### ISSUE TITLE
Question: What does `Count` mean in the Figure 8?

### ISSUE THREAD

**Issue by jasonrichdarmawan** [ApolloResearch/apd#1]
Hi, my goal is to understand Figure 8, specifically what does `Count` mean in the Figure 8?

### TLDR, scribbles

My current assumption is `Count` refers to the number of batch size, considering the secondary input feature is active as the number of batch size is increased. But:
1. It doesn't make sense, because at `Count` 1200, `APD (top-$$k$$)` has 2 MSE loss with target model output at A) $$\text{between} 10^{-5} \text{and} 10^{-4}$$ B) $$10^{-4}$$
2. the `Appendix C.2` mentioned below.

So, I don't know what `Count` stands for.

Appendix C.2:
> we use a batch size of 256

Other information:
> To investigate whether this holds true for all components and on the training data distribution, we collect MSE losses into a histogram (Figure 8).

> However, some parameter components appear to partially represent secondary input feature computations. This causes the visibly bimodal distributions of the scrubbed runs that can be seen in the figure: When these components are ablated, the loss of the model may be high when the secondary input feature is active. These components have the opposite effect on the loss when they are not ablated in the anti-scrubbed runs, making both scrubbed and anti-scrubbed losses bimodal.

![Image](https://github.com/user-attachments/assets/124ff23f-4ef3-4534-891f-3443a5fd4b9c)
